udata_datalake_service/consumer.py

```diff
@@ -8,14 +8,14 @@
 KAFKA_HOST = os.environ.get("KAFKA_HOST", "localhost")
 KAFKA_PORT = os.environ.get("KAFKA_PORT", "9092")
 KAFKA_API_VERSION = os.environ.get("KAFKA_API_VERSION", "2.5.0")
-TOPICS = os.environ.get("TOPICS", ["dataset"])
+TOPICS = os.environ.get("TOPICS", ["resource.created", "resource.modified"])
 
 
 def create_kafka_consumer():
     logging.info("Creating Kafka Consumer")
     consumer = KafkaConsumer(
         bootstrap_servers=f"{KAFKA_HOST}:{KAFKA_PORT}",
-        group_id="datalake",
+        group_id=None,
```

Comment:
Why do we set the group_id here to None ? I think group subscription to a specific group like `datalake` was a good idea.

Reply:
Indeed it's a good idea, None was useful for testing purposes and I forgot to remove it. I will do so now.